### ISSUE TITLE
Request for more animation options

### ISSUE THREAD

**Issue by mitches-got-glitches** [dai-shi/excalidraw-animate#46]
Hi, great tool - thanks for developing. There are a couple of missing animation options which I feel would elevate this tool and make it much more versatile:

+  allowing the user to select the animation speed
+ setting animation order for layers so that the user can choose which parts of their diagram are animated first and also an option to animate multiple layers concurrently

Would be good to hear if you have any ideas around this already. Thanks

**Comment by dai-shi** [comment]
https://twitter.com/dai_shi/status/1407360080568221698

**Comment by mitches-got-glitches** [comment]
Oh excellent! That sneaky hidden button. I see you have  animation order too. I've just had a little test. I was hoping that if I selected multiple layers to have the same animation order they would begin animating concurrently but that didn't seem like it was the case. Am I missing this feature too?

**Comment by dai-shi** [comment]
No, that's not implemented unfortunately.

**Comment by mitches-got-glitches** [comment]
OK, no worries. Do you think it is possible?

**Comment by dai-shi** [comment]
Technically possible, I think.
The difficulty would be how to change two behaviors/modes.
(And, that I don't have time to work on it.)

**Comment by dai-shi** [comment]
No, layers are like hidden concept in Excalidraw. By the way, they added new features since then and we might be able to make the use of it. But, everything gets too tricky.

**Comment by dai-shi** [comment]
If you haven't, try https://github.com/dai-shi/excalidraw-claymate too.

**Comment by dai-shi** [comment]
#87 ported the capability from -claymate.
